### DISCUSSION TITLE
Help me find the cause of memory leak😭

### DISCUSSION THREAD

**Discussion by zhuscat** [nuxt/nuxt#22089] [Questions]
System Info:

```
------------------------------
- Operating System: Darwin
- Node Version:     v20.4.0
- Nuxt Version:     3.6.2
- Nitro Version:    2.5.2
- Package Manager:  pnpm@8.5.1
- Builder:          vite
- User Config:      app, devServer, build, vite, vue, modules, image, css, components, i18n, pinia, imports, routeRules, experimental, appConfig, runtimeConfig
- Runtime Modules:  @nuxtjs/i18n@8.0.0-beta.13, @unocss/nuxt@0.53.5, @pinia/nuxt@0.4.11, @nuxt/image@1.0.0-rc.1, @nuxt/content@2.7.0
- Build Modules:    -
------------------------------
```

I'm finding the cause of memory leak of my Nuxt app, I have made many attempts, including update node to 20, update to latest nuxt, fix listener...

But the memory leak is still there, I take a heap snapshot and I found that `(string)` is taking up a large portion, and I found there are many http response content in `(string)`
![image](https://github.com/nuxt/nuxt/assets/14963619/053a86e0-05f0-40fe-8b14-5c0f6090fc21)

And I found there are strings like this:

![image](https://github.com/nuxt/nuxt/assets/14963619/19a08cf4-a508-4e18-9128-2d0abe404340)

![image](https://github.com/nuxt/nuxt/assets/14963619/e24a9cfe-4550-4900-900c-dd41337c33a6)

it's related to vue reactivity, I think it's supposed to be garbage collected after a request is finished?

Can someone provide me with some ideas?

😭😭😭

**Comment by zss1004504439** [comment]
same problem
